modules/upload/filetype.go

```diff
@@ -31,19 +31,16 @@ func (err ErrFileTypeForbidden) Error() string {
 func VerifyAllowedContentType(buf []byte, allowedTypes []string) error {
 	fileType := http.DetectContentType(buf)
 
-	allowed := false
 	for _, t := range allowedTypes {
 		t := strings.Trim(t, " ")
-		if t == "*/*" || t == fileType {
-			allowed = true
-			break
-		}
-	}
 
-	if !allowed {
-		log.Info("Attachment with type %s blocked from upload", fileType)
-		return ErrFileTypeForbidden{Type: fileType}
+		if t == "*/*" || t == fileType ||
+			// allowed text/plain; charset=utf-8
```

Comment:
Maybe change comment to `Allow directives after type, like 'text/plain; charset=utf-8'` to make it clear it's not only for `text/plain`.

Reply:
@silverwind I think that's reasonable and I took the liberty to push that change up.